### ISSUE TITLE
Yarn 1.x?

### ISSUE THREAD

**Issue by FezVrasta** [mhart/alpine-node#102]
When I install the container I get Yarn 0.27.5, how can I get the 1.x version?

**Comment by mhart** [comment]
Assuming you've installed nothing else in `/usr/local`, you could just do:

```console
yarn global add yarn
rm -rf /usr/local/bin/* /usr/local/share/*
```

Or you can install it manually in [the same way the Dockerfile does](https://github.com/mhart/alpine-node/blob/747a53c810f16ab76d76abc51ab2b28b268a372c/Dockerfile#L33-L41)

The next image version will have the latest yarn

**Comment by FezVrasta** [comment]
Thanks, I'll wait for the next release.

**Comment by mhart** [comment]
Just released Node v8.5.0 with Yarn 1.0.1 👍
